source/base/data_out_base.cc

```diff
@@ -828,6 +890,48 @@ namespace
       }
   }
 
+  template <int dim, int spacedim>
+  static void
+  compute_sizes(const std::vector<DataOutBase::Patch<dim, spacedim>> &patches,
+                const bool    write_higher_order_cells,
+                unsigned int &n_nodes,
+                unsigned int &n_cells,
+                unsigned int &n_points_an_n_cell)
```

Comment:
Should this be `n_points_and_n_cells`? I find the abbreviation hard to read.

Reply:
The name predates this PR but I would still like to fix it.

Reply:
I fixed this typo!